src/main/scala/hydrozoa/node/server/Api.scala

```diff
@@ -0,0 +1,30 @@
+package hydrozoa.node.server
+
+import hydrozoa.*
+import hydrozoa.infra.serializeTxHex
+
+// Types for NODE API
+
+// Head initialization
+type InitializeError = String
+
+// Simple depositing
+type DepositId = (TxId, TxIx)
+
+case class DepositRequest(
+    txId: TxId,
+    txIx: TxIx,
+    deadline: Option[BigInt],
```

Comment:
We should replace this request parameter with `depositWindow`, a non-negative Posix time duration.
```suggestion
    depositWindow: Option[BigInt],
```

The actual deposit deadline can be calculated with your formula:
```
deadline := latestBlockTime + depositMarginMaturity + depositWindow + depositMarginExpiry
```

Reply:
If the user sets the the deposit window duration too low and the head fails to absorb the deposit within the window, the deposit will be refunded immediately (if L2 consensus still holds when this is determined) or via the post-dated refund tx.

The user only loses time and tx fees.

Reply:
Of course, the frontend UI could also have additional guardrails of its own to protect the user's time and tx fees, but the hydrozoa node doesn't need to know or care about these guardrails.

Reply:
Added to #78.